pkg/tracker/tracker.go

```diff
@@ -46,19 +50,31 @@ type (
 	}
 )
 
-func init() {
+func NewTracker() error {
```

Comment:
Sorry just noticed this.
Can we please return tracker from this method ?

We should get rid of above global variable.
```
var Tracker *tracker
```

```
func NewTracker() (*tracker, error)
```

You may need to change tracker type to Tracker so that it can be exported.

So existing tracker type will become `type Tracker`

Reply:
done